move_repos_to_lgtm_lists.py

```diff
@@ -0,0 +1,39 @@
+from typing import List
+from lgtm import LGTMSite
+
+import utils.cacher
+import os
+import time
+
+def get_project_list_id(cached_file_name: str, site: 'LGTMSite') -> str:
+    project_list_name = cached_file_name.split(".")[0]
+
+    return site.get_or_create_project_list(project_list_name)
+
+def process_cached_file(cached_file_name: str, site: 'LGTMSite'):
+    cached_file = "cache/" + cached_file_name
+    project_builds = utils.cacher.get_project_builds(cached_file)
+    followed_projects = site.get_my_projects()
+
+    if project_builds.build_processes_in_progress(followed_projects):
```

Comment:
@JLLeitschuh 

You might like this little feature. We only move projects over to a custom list when all project builds are done.